codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/protocols/ec2query/EC2QueryProtocolGenerator.kt

```diff
@@ -27,6 +27,7 @@ class EC2QueryProtocolGenerator : AWSHTTPBindingProtocolGenerator(EC2QueryCustom
     override val testsToIgnore = setOf(
         "SDKAppliedContentEncoding_ec2Query",
         "SDKAppendsGzipAndIgnoresHttpProvidedEncoding_ec2Query",
+        "Ec2EmptyQueryLists",
```

Comment:
Will fix this test in a separate ticket.
https://github.com/awslabs/aws-sdk-swift/issues/1514